airbyte-ci/connectors/metadata_service/lib/metadata_service/docker_hub.py

```diff
@@ -55,7 +31,13 @@ def is_image_on_docker_hub(image_name: str, version: str) -> bool:
     Returns:
         bool: True if the image and version exists on Docker Hub, False otherwise.
     """
-    repo, image = image_name.split("/", 1)
-    tags = get_image_tags(repo, image)
 
-    return version in tags
+    token = get_docker_hub_auth_token()
+    headers = {"Authorization": f"JWT {token}"}
```

Comment:
The fact that we use the same token to connect to https://hub.docker.com makes me think https://registry.hub.docker.com is just an alias. Do you get different responses when using `registry`?

Reply:
They are different endpoints in the sense that before we were only able to get a list of tags, not individual tags.

It seems to work. but theres no way to know for sure.

We can see if we encounter the out of sync error down the road